pkg/types/hashmap/aggregate.go

```diff
@@ -86,6 +91,7 @@ func (n NamedAggFlowMapWithMetadata) Len() (l int) {
 // Clear frees as many resources as possible by making them eligible for GC
 func (n NamedAggFlowMapWithMetadata) Clear() {
 	for k, v := range n {
+		v.Stats = nil
```

Comment:
Not sure off the top of my head, but maybe this can be moved to the `v.Clear()` method, or isn't that in there? :thinking: